frontend/src/types/responses.ts

```diff
@@ -1,7 +1,10 @@
-import type { JsonRpcResult } from './results';
-
 export interface JsonRPCResponse<T> {
   id: string;
   jsonrpc: string;
-  result: JsonRpcResult<T>;
+  result: T | T[];
```

Comment:
is this line needed? since T is only used once, I think it's the same to just put T here

Reply:
done!